src/pages/campaigns/CampaignsPage.tsx

```diff
@@ -3,15 +3,15 @@ import {
   columns,
 } from "../../components/NewsletterComponents/Columns";
 import { DataTable } from "../../components/NewsletterComponents/DataTable";
-import { newsletterData } from "../../atoms/atoms";
+import { campaignData } from "../../atoms/atoms";
 import { useRecoilState } from "recoil";
 import Sidebar from "../../components/Sidebar";
 const CampaignsPage: React.FC = () => {
   useEffect(() => {
     document.title = "ACE | Campaigns";
   }, []);
 
-  const [data] = useRecoilState(newsletterData);
```

Comment:
I think here, useRecoilValue is more appropriate.

Reply:
Replaced useRecoilState with useRecoilValue in the latest commit